apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.module.css

```
@@ -0,0 +1,71 @@
+.dialog {
+  @apply py-7 px-4;
+
+  & > h1, & > h2 {
+    @apply leading-5;
+  }
+
+  & > h1 {
```

```suggestion
  .title {
```
I don't like this approach, the naming should be descriptive and not just a copy of the current CSS structure. If we decide in the future to change `h1` and `h2` to `h3` and `h4`, styles would just stop working and I can imagine it wouldn't be a pleasant debugging.